ella_flatcomments/models.py

```diff
@@ -178,3 +185,34 @@ def save(self, **kwargs):
         if self.submit_date is None:
             self.submit_date = timezone.now()
         super(FlatComment, self).save(**kwargs)
+
+    def has_edit_timer(self):
+        '''has_edit_timer() -> bool
+        '''
+        return EDIT_TIMER_ENABLED
+
+    def is_edit_timer_expired(self):
+        '''is_edit_timer_expired() -> bool
+        Check whether the comment is still within the allowed edit time
+        from the creation time.
+        '''
+        age = datetime.now(self.submit_date.tzinfo) - self.submit_date
+        if age >= timedelta(minutes=EDIT_TIMER_MINUTES):
+            return True
+        return False
+        
+    def get_remaining_edit_time(self):
```

Comment:
have this return some form of object that can be formatted (datetime of the expiration time, time or timedelta), not just string.